src/components/App.tsx

```diff
@@ -16,45 +14,50 @@ import { CurrentNodeProvider } from './CurrentNodeProvider';
 import { CurrentCriteriaProvider } from './CurrentCriteriaProvider';
 import { SnackbarProvider } from './SnackbarProvider';
 import { CriteriaBuilderProvider } from './CriteriaBuilderProvider';
+import { QueryCache, ReactQueryCacheProvider } from 'react-query';
+import StaticApp from './StaticApp/StaticApp';
```

Comment:
Add an `index.ts` in `./StaticApp` so this can be imported as `./StaticApp` similar to how we do it for other components contained in a directory